### ISSUE TITLE
Possible to keep movement mechanics while disabling player model override?

### ISSUE THREAD

**Issue by Shahelion** [makamys/SmartMoving#16]
I know it would probably look a bit janky, but would it be possible to disable the new animations while still allowing things like crawling, to allow for compatibility with mods like Custom Player Models?

**Comment by brandyyn** [comment]
Yes this is possible by using either skinport or https://github.com/jss2a98aj/BugTorch/pull/40
